.travis.yml

```diff
@@ -17,6 +17,7 @@ install:
   # Project configuration requirements.
   - pip install virtualenv virtualenvwrapper
   - pip install autoenv
+  - pip install tox
   - git clone https://github.com/cfpb/sheer
   - pip install -e sheer
   - pip install -r sheer/requirements.txt
```

Comment:
ok i'll also include sheer-like but i'm going to leave sheer since travis also tests the sheer index function